src/client/containers/Layout/index.tsx

```diff
@@ -107,9 +107,11 @@ export const Layout: FC = ({ children }) => {
   const hideControls = partner.id === 'ledger';
 
   let vaultName;
+  let titleLink;
   // TODO Add lab details route when its added the view
   if (path === 'vault') {
     vaultName = selectedVault?.displayName;
+    titleLink = '/vaults';
```

Comment:
I'm thinking that maybe all of these routes should be pulled out into a constant so that we don't need to hard code them everywhere? The `'/vaults'` route for example is used in 3 separate locations now, any thoughts? Should probably do this in a different PR though

Reply:
yeah, the most approximate think is the navigation const navLinks
```
const navLinks: NavigationLink[] = [
  {
    to: '/portfolio',
    text: 'navigation.portfolio',
    icon: WalletIcon,
  },
  {
    to: '/vaults',
    text: 'navigation.vaults',
    icon: VaultIcon,
  },
  {
    to: '/labs',
    text: 'navigation.labs',
    icon: LabsIcon,
    optional: true,
  },
  {
    to: 'https://app.ib.xyz',
    text: 'navigation.ironbank',
    icon: IronBankIcon,
    external: true,
    optional: true,
  },
  {
    to: '/settings',
    text: 'navigation.settings',
    icon: SettingsIcon,
    // hideMobile: true,
  },
];
```

Reply:
yeah agree @edwinzhng. Since right now we only have one nested route, we can just merge this one as is, with the intention of refactor it in the future since it will most likely grow.